notion/maps.py

```diff
@@ -37,6 +37,7 @@ def fget(self):
         kwargs = {}
         if "client" in signature(api_to_python).parameters:
```

Comment:
We may want to defensively check `"id"` here as well, as `api_to_python` may not have `id` in its signature

Reply:
Yes, that could be better. Do you have code suggestions for that? I'm not good at python syntaxes.

Reply:
How about this?

```python
if (
    "client" in signature(api_to_python).parameters 
    and "id" in signature(api_to_python).parameters
):
```

Reply:
Sorry I reply a bit late! Yes this looks good to me and is [what I added to my fork](https://github.com/tugboatcoding/notion-py/blob/master/notion/maps.py#L83) (I merged this PR into my fork earlier)

Reply:
Thanks all :) 
I added it